internal/pkg/service/buffer/storage/level/local/writer/test/writer.go

```diff
@@ -21,6 +21,7 @@ import (
 // Writer implements a simple writer implementing writer.Writer for tests.
 // The writer writes row values to one line as strings, separated by comma.
 type Writer struct {
+	helper  *WriterHelper
```

Comment:
Pridanim `EventWriter` sa uz jednoducho nedostanem k `test.Writer.ExpectWritesCount/TriggerSync` co sa pouziva dost v testoch.

Radsej som to vyriesil lepsie, a tieto metody vyclenil do `WriterHelper`, a tede nie potrebne riesit "pretypovanie" (interface dereference) `writer.Writer -> *test.Writer`.